### DISCUSSION TITLE
So only need the attention.py file right?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [Doggettx/stable-diffusion#1] [Q&A] [answered]
For what i read from your post if we already have the last version of main repo and in my case the webUI, i only need to replace the **attention.py** file, is that correct?

**Comment by Doggettx** [comment, marked as answer]
In principle yea, but made changes to model.py as well now. For resolutions near your limit you might get errors after rendering is done 100% with only the attention.py file.

It can still happen with the modified model.py, but should allow to go just a bit further even.

**Reply by mtweldon** [reply]
I see a few instances of model.py.. which should I be using? don't want to potentially replace the wrong one in the various folders

**Reply by Doggettx** [reply]
you can see if you check the changes, but it's in ldm/modules/diffusionmodules/model.py

Just make a copy before you replace it, just in case

**Reply by mtweldon** [reply]
ahh thank you very much

**Reply by Mightiestmike** [reply]
I'm trying to improve some abilities on my 3070 rendering, I tried copying the code from the 4 files modified but I get an error and SD won't run.... (I kept original copies so I have restored it) but can anyone walk me through what settings I need or how I go about setting up these improvements? 
I have almost no knowledge of python... I did successfully create a program that monitors the output folder from SD and uploads the files to my own private discord, but I relied HEAVILY on chatgpt for that. :p 

Anyways some guidance would be appreciated.
